ckanext/canada/tables/contracts.yaml

```diff
@@ -660,15 +660,27 @@ iii. Quand un contrat porte sur plus d’un article économique, il est recomman
         en: CFTA / CCFTA / CKFTA
         fr: ALEC / ALECC / ALECCS
       D:
-        lookup: [CETA, CFTA]
+        conditional_lookup:
+        - column: contract_date
+          less_than: '2021-01-01'
+          lookup: [CETA, TCA, CFTA]
+        - lookup: [CETA, CFTA]
```

Comment:
I think this is inverted. TCA is an agreement that applies after 2021-01-01, right?

Reply:
You're right. I just swapped them.